### ISSUE TITLE
TransferEntropy shouldn't install DynamicalSystemsBase/ChaosTools

### ISSUE THREAD

**Issue by Datseris** [JuliaDynamics/TransferEntropy.jl#55]
```
pkg> add TransferEntropy
  Updating registry at `C:\Users\datse\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed ErrorfreeArithmetic ── v0.5.0
 Installed FastRounding ───────── v0.3.1
 Installed CausalityToolsBase ─── v0.7.4
 Installed DynamicalSystemsBase ─ v1.5.3
 Installed ChaosTools ─────────── v1.9.0
 Installed Simplices ──────────── v0.5.0
 Installed DiffEqBase ─────────── v6.24.0
 Installed IntervalArithmetic ─── v0.16.1
  Updating `C:\Users\datse\ownCloud\Projects\MusicianSynchronization\Project.toml`
  [ea221983] + TransferEntropy v0.4.3
```

A lot of the above don't really make sense to be installed with this package, e.g. everything related with ODE solving: `DiffEqBase, ChaosTools, DynamicalSystemsBase`. 

I think this will be taken care of once we move all delay embedding methods (the advanced ones you have from CausalityToolsBase) to DelayEmbeddings.jl.

**Comment by kahaaga** [comment]
Thanks for the reminder! 

Yes, this will be taken care of once everything is moved to DelayEmbeddings.jl.

**Comment by kahaaga** [comment]
Closed in #58 .
